region/maridia/outer/Mt. Everest.json

```diff
@@ -990,7 +990,12 @@
         {"shinespark": {"frames": 62, "excessFrames": 8}}
       ],
       "flashSuitChecked": true,
-      "note": "Shinespark up left to get all the way up to the door.",
+      "note": [
+        "Shinespark up left to get all the way up to the door.",
+        "(This strat represents shinecharging in the room below and then diagonal shinesparking within this room. Shinesparking diagonally through the door also works.)",
+        "An easy setup is to spinjump up and facing left into the door transition.  During the transition, release all other controls and hold jump + angle aim up.",
+        "Samus will immediately shinepspark."
+      ],
       "devNote": "Coming in shinesparking can get Samus to 11. With Gravity, that has fewer requirements."
```

Comment:
not entirely sure what this note means

Reply:
I'm not sure what this means either.  A [2,11] through-door shinespark strat exists.  This can probably be deleted.
Or it could mean you can shinespark elsewhere.  Because you can reach 10, or 9 with a suitless comeInShinecharged.